openwisp_network_topology/api/views.py

```diff
@@ -75,7 +77,16 @@ def post(self, request, pk, format=None):
                 'with message "%s"'
             ) % (topology.get_parser_display(), e.__class__.__name__, e)
             return Response({'detail': error}, status=400)
-        return Response({'detail': _('data received successfully')})
+        success_message = _('data received successfully')
+        deprecated_url = f'/api/v1/receive/{pk}/'
+        if request.path == deprecated_url:
+            warning = _(
+                "This URL is depercated and will be removed in "
+                "future versions, use /topology/{id}/receive/"
+            )
+            logger.warning(warning)
+            success_message = f'{success_message}. {warning}'
+        return Response({'detail': _(success_message)})
```

Comment:
should be `success_message` instead of `_(success_message)` because it was already marked as translatable before